app/views/shared/_attributes.html.erb

```diff
@@ -1,8 +1,11 @@
-      <table class="table">
+<% include_title ||= false # title is part of the exterior template in common cases %>
```

Comment:
My philosophy is not to build software that nobody is asking for. All code has bugs and thus the less code we build the higher quality product we have. Doubly so for untested code.